usr/bin/cylc

```diff
@@ -191,7 +191,6 @@ elif [[ -d "${CYLC_HOME}/conda-meta" && \
         -f "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh" ]]; then
     # A normal Conda environment
     . "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh"
-    conda activate "${CYLC_HOME##*/}" || exit 1
```

Comment:
Can we have a comment in the file for future reference, on why we're not using `conda activate` and why that is OK?

Reply:
Note there's an existing `conda activate` comment and variable above lines 180-4, which is presumably now redundant.

Reply:
Do we need to do the unset below it?

Reply:
No - it was only needed due to the "conda activate"